google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/Publisher.java

```diff
@@ -827,6 +855,22 @@ public Builder setEndpoint(String endpoint) {
       return this;
     }
 
+    /** Gives the ability to enable transport compression. */
+    public Builder setEnableCompression(boolean enableCompression) {
```

Comment:
I am not sure if this is considered a "BetaApi" feature or not, but if it is, can we add the `BetaApi` annotation?

Similar to https://github.com/googleapis/java-pubsub/blob/main/google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/AckReplyConsumerWithResponse.java#L36-L37

Reply:
@rajanya-google TBD next Wed.

Reply:
This feature is going straight to GA.